index.js

```diff
@@ -1,8 +1,29 @@
-var https = require('https');
+var request = require('request');
 var readline = require('readline'),
     rl = readline.createInterface(process.stdin, process.stdout);
 const fs = require('fs');
 
+if (process.argv.length <= 2) {
+    console.log('Usage: master password needs to be passed on command line');
+    process.exit(-1);
+}
```

Comment:
I would prefer that we prompt for the master password, it's not a good practice to set a password in the command line as we can find it easily with for example `ps`